core/shared/src/main/scala/fs2/Stream.scala

```diff
@@ -2309,7 +2309,7 @@ final class Stream[+F[_], +O] private[fs2] (private val free: FreeC[F, O, Unit])
   )(f: (O2, O3) => O4): Stream[F2, O4] =
     this.parZip(that).map(f.tupled)
 
-  /** Pause this stream when `pauseWhenTrue` emits `False`, resuming when `True` is emitted. */
+  /** Pause this stream when `pauseWhenTrue` emits `true`, resuming when `false` is emitted. */
```

Comment:
Perhaps it's also worth mentioning that this stream will be interrupted if the `pauseWhenTrue` stream terminates.